pandas/tests/io/parser/common.py

```diff
@@ -1695,3 +1695,40 @@ class InvalidBuffer(object):
 
             with tm.assertRaisesRegexp(ValueError, msg):
                 self.read_csv(mock.Mock())
+
+    def test_skip_bad_lines(self):
+        data = 'a\n1\n1,2,3\n4\n5,6,7'
+
+        with tm.assertRaises(ParserError):
+            self.read_csv(StringIO(data))
+
+        with tm.assertRaises(ParserError):
+            self.read_csv(StringIO(data), error_bad_lines=True)
+
+        stderr = sys.stderr
+        expected = DataFrame({'a': [1, 4]})
```

Comment:
That's possible, though not sure how if that would integrate well with `pytest` 's fixture setup that I mentioned <a href="https://github.com/pandas-dev/pandas/pull/15925#discussion_r110310985">earlier</a>.